locales/en-US/server.properties

```diff
@@ -0,0 +1,146 @@
+# global phrases shared across pages, prefixed with 'g'
+
+# TODO: replace each page's reference to this with the global
+gMyShots = My Shots
+
+# creating page
+
+# Localization note: {title} is a placeholder for the title of the web page
+# captured in the screenshot. The default, for pages without titles, is
+# creatingPageTitleDefault.
+creatingPageTitle = Creating {title}
+creatingPageTitleDefault = page
+
+# home page
+
+# Note: do not translate the phrase 'Firefox Screenshots'.
+homePageDescription = Share anything on the web with anyone using Firefox Screenshots.
+# TODO: do we need this separate description meta tag?
+homePageDescription2 = Intuitive screenshots baked right into the browser. Capture, save and share screenshots as you browse the Web using Firefox.
+homePageButtonMyShots = Go To My Shots
+homePageTitleBeta = Beta
+homePageTeaser = Coming Soon...
+
+# leave screenshots page
+
+leavePageConfirmDelete = Confirm account deletion
+leavePageErrorAddonRequired = You must have the addon installed to delete your account
+# TODO: do we need this separate error message?
+leavePageErrorAddonRequired2 = You must have the addon installed to leave
+leavePageErrorGeneric = An error occurred
+leavePageWarning = This will permanently erase all of your Firefox Screenshots data.
+leavePageButtonProceed = Proceed
+leavePageButtonCancel = cancel
+# Note: do not translate the phrase 'Firefox Screenshots' :-)
+leavePageDeleted = All of your Firefox Screenshots data has been erased.
+
+# metrics page
+
+# Note: 'Firefox Screenshots' should not be translated
+metricsPageTitle = Firefox Screenshots Metrics
+
+# Note: {status} is a placeholder for an HTTP status number, like 403 or 500.
+metricsPageAlertFailure = Failed: {status}
+metricsPageTotalsQueryTitle = Totals
+metricsPageTotalsQueryDescription = Various totals from the database
+metricsPageTotalsQueryDevices = Total devices registered
+metricsPageTotalsQueryActiveShots = Active shots
+metricsPageTotalsQueryExpiredShots = Expired (recoverable)
+metricsPageTotalsQueryExpiredDeletedShots = ... (deleted)
+metricsPageShotsQueryTitle = Shots By Day
+metricsPageShotsQueryDescription = Number of shots created each day (for the last 30 days)
+metricsPageShotsQueryCount = Number of shots
+metricsPageShotsQueryDay = Day
+metricsPageUsersQueryTitle = Users By Day
+metricsPageUsersQueryDescription = Number of users who created at least one shot, by day (last 30 days)
+metricsPageUsersQueryCount = Number of users
+metricsPageUsersQueryDay = Day
+metricsPageUserShotsQueryTitle = Number of Shots per User
+metricsPageUserShotsQueryDescription = The number of users who have about N total shots
```

Comment:
Yup, where N is an integer. The wording for these metrics strings could definitely use some help from non-engineers :-)